src/tooltip/tooltip.js

```diff
@@ -260,6 +260,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
              * Observe the relevant attributes.
              */
             attrs.$observe( type, function ( val ) {
+              if (val && val.length >= 1 && val.charAt(0) == '@') {
+                var tag = element.find(val.length > 1 ? val.substring(1) : 'tooltip');
+                if (tag.length == 1) {
+                  ttScope.content = tag.html();
+                  tag.remove();
+                  return;
+                }
```

Comment:
This is all better broken up as
<pre>
var tag;
if (val.length > 1) {
  tag = element.find(val.substr(1));
  ttScope.content = tag.html();
  return tag.remove();
}

tag = element.find('tooltip');
</pre>